docs/manuals/gui/viselements/charts/scatter.md

```diff
@@ -1,9 +1,24 @@
-## Scatter charts
+# Scatter charts
 
 Scatter charts represent data points as dots or other symbols.
 They are useful to see relationships between groups of data points.
 
-### Classification
+In order to create scatter charts with Taipy, you need to set the
+[_mode[]_](../chart.md#p-mode) property for your trace to "markers".
+
+## Key properties
+
+| Parameter       | Value             | Notes |
+| --------------- | ------------------------- | ------------------ |
+| [_mode_](../chart.md#p-mode)      | `markers`          |  |
+| [_x_](../chart.md#p-x)            | x values           |  |
+| [_y_](../chart.md#p-y)            | y values           |  |
+| [_marker_](../chart.md#p-marker)  | dictionary  | `size` can be an integer or a column name in _data_.<br/>`color` can be a color value or a column name in _data_.<br/>`symbol` must be a predefined symbol name. |
```

Comment:
"color", "size", "symbol", "opacity" can be column names

Reply:
👍